modules/social_features/social_core/social_core.install

```diff
@@ -506,3 +506,84 @@ function social_core_update_130006(): void {
   // updated to ignore those files, this causes them to error.
   \Drupal::database()->query("UPDATE {file_managed} SET uri = REPLACE(uri, 'secret://', 'private://') WHERE uri LIKE 'secret://inline-images/%'");
 }
+
+/**
+ * Revert field_files to use private:// instead of secret://.
+ */
+function social_core_update_130007(): void {
+  // Change storage uri-scheme to private for file-field.
+  $file_fields = \Drupal::entityTypeManager()
+    ->getStorage('field_storage_config')
+    ->loadByProperties([
+      'type' => 'file',
+    ]);
+
+  foreach ($file_fields as $field_storage) {
+    if (
+      $field_storage->get('field_name') === 'field_files'
+      && $field_storage->getSetting('uri_scheme') === 'secret'
+    ) {
+      $field_storage->setSetting('uri_scheme', 'private');
+      $field_storage->save();
+    }
+  }
+
+  // Change file_managed table only files from field_file field.
+  // Get all file-id and load them.
+  $files_id = \Drupal::entityTypeManager()
+    ->getStorage('file')
+    ->getQuery()
+    ->accessCheck(FALSE)
+    ->execute();
+  $files = \Drupal::entityTypeManager()
+    ->getStorage('file')
+    ->loadMultiple($files_id);
```

Comment:
Instead of doing a file load and looping through that, we can simply use a single database query!

As a normal SQL query it'll look like this (test it to see how it works):

```sql
UPDATE file_managed
SET uri = REPLACE(uri, "secret://", "private://")
WHERE fid IN (SELECT field_files_target_id FROM node__field_files)
```

Instead of doing entity loads we just query for all File IDs from the `node__field_files` table and use that to update the scheme from `secret://` to `private://`.

Reply:
This is also very quick, and as we can run the query targeted (we only need to update one file field), it's also pretty safe :)